### ISSUE TITLE
Tab component tests

### ISSUE THREAD

**Issue by mjcarsjens** [nl-design-system/denhaag#425]
The current implementation of the tab component spec is rather lackluster. It currently does a visual, html snapshot and a11y test, but no functionality tests are performed due to troubles with making React state work in cypress. Requires some more reading into to find out how to properly implement this, and then it needs to be implemented.
